src/Quizzer.API.BL/Models/AnswerDetailModel.cs

```diff
@@ -6,8 +6,6 @@ public class AnswerDetailModel : IModel {
 
         public AnswerFormat Type { get; set; }
 
-        public Guid QuestionId { get; set; } 
```

Comment:
i do not understand why?

Reply:
> i do not understand why?

@KoubaFromPucks Because I think it is not necessary now as I introduced `AddAnswer` method. Do you have opposite opinion?

Reply:
now no